event-adapter/README.md

```diff
@@ -0,0 +1,181 @@
+# 1. Create a cluster ensuring that Stackdriver Logging API are enabled (both reading and writing)
+```sh
+mkdir -p $GOPATH/src/k8s.io
+cd $GOPATH/src/k8s.io
+git clone git@github.com:kubernetes/kubernetes.git
+cd kubernetes
+git checkout 9aef242a4c1e423270488206670d3d1f23eaab52
+sed -i 's/logging-write/https:\/\/www.googleapis.com\/auth\/logging.admin/g' cluster/gce/config-common.sh
+make quick-release
+./cluster/kube-up.sh
+```
+You should create “k8s.io” folder to host the project. Then, clone kubernetes project on git. Checkout to the desired version and compile it.
+Stackdriver Logging API has only write permission on the default version, but we need both reading and writing.
+So, you can modify "kubernetes/cluster/gce/config-common.sh" replacing "logging-write" with "https://www.googleapis.com/auth/logging.admin".
+Now, you can create the cluster.
+Call "kube-up.sh"script, inside “cluster” folder,  to create the cluster.
+
+
+# 2. Install the API
+
+Clone the project "https://github.com/GoogleCloudPlatform/k8s-stackdriver" into your GOPATH.
+```sh
+mkdir -p $GOPATH/src/github.com/GoogleCloudPlatform
+cd $GOPATH/src/github.com/GoogleCloudPlatform
+git clone git@github.com:GoogleCloudPlatform/k8s-stackdriver.git
+git checkout 692c11f5395b3b1d321d8384cde56a4112b5696d
+cd k8s-stackdriver/event-adapter
+```
+Now, you need to set it to work with your project. You should define a variable that identify your project GCP_PROJECT=myProject.
+```sh
+sed -i 's/google-containers/'"$GCP_PROJECT"'/g’adapter.yaml
+make push GCP_PROJECT
+kubectl create -f adapter.yaml --validate=false
+```
+The adapter.yaml file contains the destination from which you will retrieve the image, Makefile instead will contain the place where you are going to push the image.
```

Comment:
"The adapter.yaml file contains the destination from which you will retrieve the image" -> "The adapter.yaml file specifies the source of Event Adapter Docker image"

"Makefile instead will contain the place where you are going to push the image" -> "and Makefile describes the destination where the image you build is going to be pushed"

Reply:
Done